src/measurement/gctx.rs

```diff
@@ -12,6 +13,14 @@ use crate::{
     measurement::snp::{SnpLaunchDigest, LD_BYTES},
 };
 
+#[cfg(all(not(feature = "openssl"), feature = "crypto_nossl"))]
```

Comment:
The library won't compile if both crypto_nossl and openssl are both passed as features, so you can just gate under crypto_nossl

```suggestion
#[cfg(feature = "crypto_nossl")]
```

Reply:
this is like this to prevent two sha384 to be in scope if both openssl  and crypto_nossl is enabled

Reply:
This was my mistake, apologies @jakubDoka